ajax/libs/mmenu-js/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "mmenu-js",
+  "filename": "mmenu.min.js",
```

Comment:
```suggestion
  "filename": "mmenu.js",
```

Looking at the latest release on NPM, a minified version is not included in the `dist` directory:

```
node_modules/mmenu-js/dist
├── _modules
│   ├── dom.js
│   ├── dragevents
│   │   ├── _defaults.js
│   │   ├── _helpers.js
│   │   ├── _settings.js
│   │   ├── _support.js
│   │   └── index.js
│   ├── eventlisteners.js
│   ├── helpers.js
│   ├── i18n.js
│   ├── matchmedia.js
│   └── support.js
├── _version.js
├── addons
│   ├── autoheight
│   │   ├── _options.js
│   │   ├── mmenu.autoheight.css
│   │   └── mmenu.autoheight.js
│   ├── backbutton
│   │   ├── _options.js
│   │   └── mmenu.backbutton.js
│   ├── columns
│   │   ├── _options.js
│   │   ├── mmenu.columns.css
│   │   └── mmenu.columns.js
│   ├── counters
│   │   ├── _options.js
│   │   ├── mmenu.counters.css
│   │   └── mmenu.counters.js
│   ├── dividers
│   │   ├── _options.js
│   │   ├── mmenu.dividers.css
│   │   └── mmenu.dividers.js
│   ├── drag
│   │   ├── _drag.open.js
│   │   ├── _options.js
│   │   ├── mmenu.drag.css
│   │   └── mmenu.drag.js
│   ├── dropdown
│   │   ├── _configs.js
│   │   ├── _options.js
│   │   ├── mmenu.dropdown.css
│   │   └── mmenu.dropdown.js
│   ├── fixedelements
│   │   ├── _configs.js
│   │   └── mmenu.fixedelements.js
│   ├── iconbar
│   │   ├── _options.js
│   │   ├── mmenu.iconbar.css
│   │   └── mmenu.iconbar.js
│   ├── iconpanels
│   │   ├── _options.js
│   │   ├── mmenu.iconpanels.css
│   │   └── mmenu.iconpanels.js
│   ├── keyboardnavigation
│   │   ├── _options.js
│   │   ├── mmenu.keyboardnavigation.css
│   │   └── mmenu.keyboardnavigation.js
│   ├── lazysubmenus
│   │   ├── _options.js
│   │   └── mmenu.lazysubmenus.js
│   ├── navbars
│   │   ├── _configs.js
│   │   ├── _navbar.breadcrumbs.js
│   │   ├── _navbar.close.js
│   │   ├── _navbar.next.js
│   │   ├── _navbar.prev.js
│   │   ├── _navbar.searchfield.js
│   │   ├── _navbar.tabs.js
│   │   ├── _navbar.title.js
│   │   ├── _options.js
│   │   ├── mmenu.navbars.css
│   │   └── mmenu.navbars.js
│   ├── pagescroll
│   │   ├── _configs.js
│   │   ├── _options.js
│   │   └── mmenu.pagescroll.js
│   ├── searchfield
│   │   ├── _configs.js
│   │   ├── _options.js
│   │   ├── mmenu.searchfield.css
│   │   ├── mmenu.searchfield.js
│   │   └── translations
│   │       ├── de.js
│   │       ├── fa.js
│   │       ├── nl.js
│   │       ├── ru.js
│   │       └── translate.js
│   ├── sectionindexer
│   │   ├── _options.js
│   │   ├── mmenu.sectionindexer.css
│   │   └── mmenu.sectionindexer.js
│   ├── setselected
│   │   ├── _options.js
│   │   ├── mmenu.setselected.css
│   │   └── mmenu.setselected.js
│   ├── sidebar
│   │   ├── _options.js
│   │   ├── mmenu.sidebar.css
│   │   └── mmenu.sidebar.js
│   └── toggles
│       ├── mmenu.toggles.css
│       └── mmenu.toggles.js
├── core
│   ├── offcanvas
│   │   ├── _configs.js
│   │   ├── _options.js
│   │   ├── mmenu.offcanvas.css
│   │   └── mmenu.offcanvas.js
│   ├── oncanvas
│   │   ├── _configs.js
│   │   ├── _options.js
│   │   ├── mmenu.oncanvas.css
│   │   ├── mmenu.oncanvas.js
│   │   └── translations
│   │       ├── de.js
│   │       ├── fa.js
│   │       ├── nl.js
│   │       ├── ru.js
│   │       └── translate.js
│   ├── screenreader
│   │   ├── _configs.js
│   │   ├── _options.js
│   │   ├── mmenu.screenreader.css
│   │   ├── mmenu.screenreader.js
│   │   └── translations
│   │       ├── de.js
│   │       ├── fa.js
│   │       ├── nl.js
│   │       ├── ru.js
│   │       └── translate.js
│   └── scrollbugfix
│       ├── _options.js
│       └── mmenu.scrollbugfix.js
├── extensions
│   ├── borderstyle
│   │   └── mmenu.borderstyle.css
│   ├── effects
│   │   └── mmenu.effects.css
│   ├── fullscreen
│   │   └── mmenu.fullscreen.css
│   ├── listview
│   │   └── mmenu.listview.css
│   ├── multiline
│   │   └── mmenu.multiline.css
│   ├── pagedim
│   │   └── mmenu.pagedim.css
│   ├── popup
│   │   └── mmenu.popup.css
│   ├── positioning
│   │   └── mmenu.positioning.css
│   ├── shadows
│   │   └── mmenu.shadows.css
│   ├── themes
│   │   └── mmenu.themes.css
│   └── tileview
│       └── mmenu.tileview.css
├── mmenu.css
├── mmenu.js
├── mmenu.polyfills.js
└── wrappers
    ├── angular
    │   └── mmenu.angular.js
    ├── bootstrap
    │   ├── mmenu.bootstrap.css
    │   └── mmenu.bootstrap.js
    ├── magento
    │   └── mmenu.magento.js
    ├── olark
    │   └── mmenu.olark.js
    ├── turbolinks
    │   └── mmenu.turbolinks.js
    └── wordpress
        └── mmenu.wordpress.js
```